WhaleSpotting/react-app/src/styles/Profile.scss

```diff
@@ -0,0 +1,117 @@
+@import './Constants.scss';
+
+.body {
+    width: 100vw;
+    min-width: $minMobileScreenSize;
+    font-family: $fontFamily;
+    display: flex;
+    flex-direction: column;
+}
```

Comment:
I'd have new lines in between each class

Reply:
Addressed.